docs/standard/modernize-with-azure-and-containers/modernize-existing-apps-to-cloud-optimized/modernize-your-apps-lifecycle-with-ci-cd-pipelines-and-devops-tools-in-the-cloud.md

```diff
@@ -13,8 +13,6 @@ Azure DevOps Services supports continuous integration and deployment of multi-co
 
 - [Deploy to standalone Docker Host VM](https://docs.microsoft.com/azure/devops/build-release/apps/cd/deploy-docker-windowsvm) (Linux or Windows Server 2016 or later)
```

Comment:
This link has changed to https://docs.microsoft.com/en-us/azure/devops/pipelines/apps/cd/deploy-docker-webapp?view=azure-devops 
Replace text `Deploy to standalone Docker Host VM (Linux or Windows Server 2016 or later)` to `Deploy to an Azure Web App for Containers`